rs-client-demo/src/main.rs

```diff
@@ -12,6 +13,7 @@ impl Verification for CustomVerifier {
 
 #[tokio::main]
 async fn main() {
+    env_logger::init();
```

Comment:
What does this do? If logging can be configured through environment, then it should be documented somewhere (README?)

Reply:
env_logger::init() is required to initialize the global logger; sure I will add it to the README